src/user-management/user-management.ts

```diff
@@ -79,6 +79,22 @@ import {
 } from './interfaces/send-invitation-options.interface';
 import { serializeSendInvitationOptions } from './serializers/send-invitation-options.serializer';
 import { serializeListUsersOptions } from './serializers/list-users-options.serializer';
+import { AuthorizationURLOptions } from './interfaces/authorization-url-options.interface';
+
+const toQueryString = (options: Record<string, string | undefined>): string => {
+  const searchParams = new URLSearchParams();
+  const keys = Object.keys(options).sort();
+
+  for (const key of keys) {
+    const value = options[key];
+
+    if (value) {
+      searchParams.append(key, value);
+    }
+  }
+
+  return searchParams.toString();
+};
```

Comment:
We should probably extract this into a shared helper under `src/common/utils/to-query-string.ts`, though we can do that later.